packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -79,12 +79,12 @@ function isCreateNodeEvent(
 }
 
 function isMenuItemEnabled(menu: ContextMenu, action: string) {
-  const item = menu.find((m) => {
+  const item: any = menu.find((m: any) => {
```

Comment:
don't use any

Reply:
I started using `ContextMenuItem` type but got the error:
```
(rpt2 plugin) Error: /packages/pipeline-editor/src/PipelineEditor/index.tsx(83,9): semantic error TS2322: Type 'ContextMenuItem | undefined' is not assignable to type 'ContextMenuItem'.
  Type 'undefined' is not assignable to type 'ContextMenuItem'.
```

Reply:
I think you would just need to check for undefined then. I'll pull this and take a closer look